### PR TITLE
Release logging 1.8.0

### PR DESCRIPTION
**Pull request by crwilcox**
This pull request was generated using releasetool.

### CODE REVIEW

File: logging/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
 
 [1]: https://pypi.org/project/google-cloud-logging/#history
 
+## 1.8.0
+
+### New Features
+- Logging: support request-correlated logging in App Engine standard python37 runtime ([#6118](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6118))
```
Comment (tseaver) [inline logging/CHANGELOG.md]: Delete the `Logging:` prefix.

**Comment by crwilcox** [comment]
Seems this change may have introduced an issue for our system tests. The first run failed sys27 and the next failed sys27 and sys36. I am going to force run again to be sure.

**Comment by tseaver** [comment]
@crwilcox See the list of [flaky logging system test issues](https://github.com/googleapis/google-cloud-python/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3Aflaky+label%3A%22api%3A+logging%22+).

**Comment by tseaver** [comment]
I'd like to re-do this release after closing all the flaky systest issues.

**Pull request closed without merging**